qiskit-qir/setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = qiskit-qir
+version = 0.0.1
+author = Microsoft
+author_email = que-contacts@microsoft.com
+description = Qiskit to QIR translator
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/microsoft/qdk-python
+project_urls =
+    Bug Tracker = https://github.com/microsoft/qdk-python
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
```

Comment:
@guenp - given that pyqir is platform specific - is this lib really OS Independent?

Reply:
I've added more specific classifiers.